tools/src/commands/ValidateWorkspaceDependencies.ts

```diff
@@ -0,0 +1,75 @@
+import { Command } from '@expo/commander';
+import path from 'path';
+
+import { getExpoRepositoryRootDir } from '../Directories';
+import { getInfoAsync } from '../Workspace';
+
+type ActionOptions = {
+  workspace?: string;
+};
+
+export default (program: Command) => {
+  program
+    .command('validate-workspace-dependencies')
+    .alias('vwd')
+    .description('Verifies if all workspaces are linked from monorepo.')
+    .asyncAction(action);
+};
+
+async function action(options: ActionOptions) {
+  const workspaces = await getWorkspacesAsync();
+  const workspacesMismatched = Object.values(workspaces).filter(
+    (workspace) => workspace.mismatchedWorkspaceDependencies.length
+  );
+
+  if (!workspacesMismatched.length) {
+    return console.log(`✅  Verified that all workspace dependencies are symlinked.`);
```

Comment:
This seems a bit odd, I prefer not to return something from another function call when we're expecting it to return `undefined`

```suggestion
    console.log(`✅  Verified that all workspace dependencies are symlinked.`);
    return
```